backend/rates/history_test.go

```diff
@@ -42,9 +42,7 @@ func TestPriceAt(t *testing.T) {
 		{7.25, time.Date(2020, 9, 3, 18, 0, 0, 0, time.UTC)},
 	}
 	for _, test := range tt {
-		assert.Equal(t,
-			test.wantValue,
-			updater.HistoricalPriceAt("btc", "USD", test.at), "at = %s", test.at)
+		assert.InDelta(t, test.wantValue, updater.HistoricalPriceAt("btc", "USD", test.at), 0, "at = %s", test.at)
```

Comment:
This is an instance of making code less readable to please the linter. Can the `InDelta` check only be disabled?